src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -27,7 +28,13 @@ public static void printLongestWordTop100() throws IOException {
                 .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
-        // TODO 이 부분에 구현한다.
+        words.stream()
```

Comment:
```suggestion
        words.stream()
                .filter(s -> s.length() > 12)
                .distinct()
                .limit(100L)
                .sorted(Comparator.comparing(String::length).reversed())
                .map(String::toLowerCase)
                .forEach(System.out::println);
```
limit을 정렬하는 작업 위에 두면 어떨까요? limit 전까지는 모든 값을 가지고 정렬을 하게 될텐데 100개의 유니크한 단어들만 가지고 정렬해도 동일한 결과가 나올거라 불필요한 작업을 줄이면 좋을거 같아 의견 드립니다.